### DISCUSSION TITLE
Create a TypeScript declaration file

### DISCUSSION THREAD

**Discussion by stretch07** [rigwild/discord-self-bot-console#39] [General]
hi @rigwild 
not sure if you've had a lot of experience with typescript, but i think this would be beneficial to this project. Especially in IDEs where not knowing types bothers both me and the editor.
is this something you would like to implement, me to implement (i'd be glad to do it) or not to be implemented at all?
:)

**Comment by DeadLyBro** [comment]
Is it possible to do this? @rigwild 
or is it out of this project 😄

**Comment by rigwild** [comment]
Good idea! #60

**Reply by stretch07** [reply]
Awesome! Thanks
